.github/workflows/whitesource.yml

```
@@ -0,0 +1,33 @@
+name: Whitesource Unified Agent Scan - GO
+
+on:
+  pull_request:
+    branches: [ master* ]
+
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.12.17
```

```suggestion
        go-version: 1.17
 ```
 
 The go.mod was originally made with 1.12, but if you look at our build matrix, we only build against 1.14 - 1.17 now. We can update the go.mod later to reflect this change